seqr/views/utils/variant_utils.py

```diff
@@ -66,13 +66,22 @@ def update_projects_saved_variant_json(projects, user_email, **kwargs):
     return updated_variants_by_id
 
 
-def update_project_saved_variant_json(project_id, family_guids=None, dataset_type=None, user=None, user_email=None):
-    saved_variants = SavedVariant.objects.filter(family__project_id=project_id).select_related('family')
+def get_saved_variants(genome_version, project_id=None, family_guids=None, dataset_type=None):
+    saved_variants = SavedVariant.objects.filter(
+        Q(saved_variant_json__genomeVersion__isnull=True) |
+        Q(saved_variant_json__genomeVersion=genome_version.replace('GRCh', ''))
```

Comment:
you could pass `db_genome_version` to this function and use it here instead of calling `replace('GRCh', ''))` again

Reply:
yeah but then in the other place its used I would have to call replace, which currently is not the case